app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_results = {}
+
+    def wrapper(*args: (int, float, bool, str, bytes, tuple)) -> Any:
```

Comment:
There is no need to use type annotation for `args` and `kwargs`, remove it from the function signature.